pkg/profiler/rlimit.go

```diff
@@ -68,3 +68,17 @@ func HumanizeRLimit(val uint64) string {
 	}
 	return humanize.Bytes(val)
 }
+
+// Files returns the currently opened file descriptors as well
+// as the maximum number of file descriptors that can be
+// opened by the calling process.
+func Files() (int, int, error) {
```

Comment:
nit: I think at this point, we should have a `pkg/rlimit` package. This is already used by others part of the system other than `pkg/profiler`

Reply:
Sounds good!